spec/controllers/customer_requests_controller_spec.rb

```diff
@@ -24,9 +24,14 @@
   end
 
   describe 'POST #create' do
-    it '' do
+    it 'creates and saves a new customer request to the database' do
+      # CustomerRequest.new = [
+      post :create, customer_request: attributes_for(:customer_request)
+      expect(CustomerRequest.count).to eq(1)
```

Comment:
you want to see the count increase by 1, this won't always equal 1 because the db isn't wiped between tests